### PR TITLE
Timeline selection dropdown PR

### PR DESCRIPTION
**Pull request by Joy-Adah**
Added a disabled dropdown to the header with a 7 days label.

### CODE REVIEW

**Review by melnikga — CHANGES_REQUESTED** [review]
Hi, there is no need to make up the Select component from scratch. Use ready-made components from shadcn. There is a component called Select.

File: src/components/header.tsx [hunk 1]
```diff
@@ -18,8 +25,21 @@ export default function Header() {
...
+          <div className="relative inline-block group">
+            <Select>
+              <SelectTrigger className="w-[180px]" disabled>
+                <SelectValue placeholder="7 days" />
+              </SelectTrigger>
+              <SelectContent>
+                <SelectItem value="7">7 days</SelectItem>
+              </SelectContent>
+            </Select>
+            <div className="absolute top-full left-1/2 transform -translate-x-1/2 mt-2 w-48 bg-gray-700 text-white text-center rounded-lg py-2 opacity-0 transition-opacity duration-300 group-hover:opacity-100 pointer-events-none text-sm">
```
Comment (melnikga) [inline src/components/header.tsx:39]: Import tolltip from shadcn

**Review by melnikga — CHANGES_REQUESTED** [review]
Hi, the tooltip also needs to be imported from shadcn

**Review by melnikga — CHANGES_REQUESTED** [review]
Hi, when the project starts, errors appear that are visible in the console and displayed on the page. This is related to incorrect nesting of components and the absence of the 'asChild' attribute in TooltipTrigger. The last screenshot shows the correct structure.
Error:
image
The right structure:
image

**Comment by Joy-Adah** [comment]
Hi @melnikga I'll have it implemented

**Comment by Joy-Adah** [comment]
hi @melnikga i have implemented the suggested changes

**Review by melnikga — APPROVED** [review]
LGTM!

**Comment by melnikga** [comment]
@Joy-Adah resolve branch conflicts please

**Review by barabanovro — APPROVED** [review]

**Pull request merged**